FtoC.java

```diff
@@ -1,13 +1,24 @@
 class FtoC extends ConsoleProgram {
 
   /**
-  * Description
-  * @author:
+  * Coverting F to C
+  * @author: Alina Meng
   */
   
   public void run() {
     
-    // start coding here
+    // Variables
+    double dblFarTemp;
+    double dblCelTemp;
+
+    // user input
+    dblFarTemp = readDouble("What is the temperature in farenheit? ");
+
+    // calc
+    dblCelTemp = (5*(dblFarTemp - 32)) / 9;
```

Comment:
expanded the equation instead of factoring out the 5/9 so there isn't a problem in the calculation.